.pipeline/blubber.yaml

```diff
@@ -0,0 +1,29 @@
+version: v3
+base: docker-registry.wikimedia.org/nodejs-slim
+lives:
+  in: /srv/service
+runs:
+  environment: { APP_BASE_PATH: /srv/service }
+
+variants:
+  build:
+    base: docker-registry.wikimedia.org/nodejs-devel
+    apt: { packages: [git, python-pkgconfig, build-essential] }
```

Comment:
Are there any native nodejs extensions in restbase? Otherwise I think build-essential and python-pkgconfig are not required

Reply:
Only for prod stuff, the `heapdump` and `gc-stats`, but I don't know if that's already accounted for or not?

Reply:
Good point, I 'll have to look into it

Reply:
It seems we do need them after all.

Reply:
gc-stats seems to not require it, but heapdump does. So let's leave it as is.